pkg/pillar/dpcmanager/dpcmanager.go

```diff
@@ -117,7 +117,9 @@ type DpcManager struct {
 	dpcTestBetterInterval time.Duration // Look for lower/better index
 	geoRedoInterval       time.Duration
 	geoRetryInterval      time.Duration
+	enableLastResort      bool
```

Comment:
Again, not important, but I would put this attribute [here](https://github.com/lf-edge/eve/blob/master/pkg/pillar/dpcmanager/dpcmanager.go#L89-L94). This place here is for time-related things :)

Reply:
Done